pkgs/applications/audio/radiotray-ng/default.nix

```diff
@@ -0,0 +1,87 @@
+{ stdenv, fetchFromGitHub, cmake, pkgconfig
+# Transport
+, curl
+# Libraries
+, boost
+, jsoncpp
+, libbsd
+, pcre
+# GUI/Desktop
+, glibmm
+, libappindicator-gtk3
+, libnotify
+, libxdg_basedir
+# GStreamer
+, gst_all_1
+# User-agent info
+, lsb-release
+# rt2rtng
+, python2
+# Testing
+, gmock
+# Fixup
+, wrapGAppsHook
+, makeWrapper
+}:
+
+let
+  gstInputs = with gst_all_1; [
+    gstreamer gst-plugins-base
+    gst-plugins-good gst-plugins-bad gst-plugins-ugly
+    gst-libav
+  ];
+  # For the rt2rtng utility for converting bookmark file to -ng format
+  pythonInputs = with python2.pkgs; [ python2 lxml ];
+in
+stdenv.mkDerivation rec {
+  name = "radiotray-ng-${version}";
+  version = "0.1.7";
+
+  src = fetchFromGitHub {
+    owner = "ebruck";
+    repo = "radiotray-ng";
+    rev = "v${version}";
+    sha256 = "1m853gzh9r249crn0xyrq22x154r005j58b0kq3nsrgi5cps2zdv";
+  };
+
+  nativeBuildInputs = [ cmake pkgconfig wrapGAppsHook makeWrapper ];
+
+  buildInputs = [
+    curl
+    boost jsoncpp libbsd pcre
+    glibmm libappindicator-gtk3 libnotify
+    libxdg_basedir
+    lsb-release
+  ] ++ stdenv.lib.optional doCheck gmock
+    ++ gstInputs
+    ++ pythonInputs;
+
+  prePatch = ''
+    substituteInPlace debian/CMakeLists.txt \
+      --replace /usr $out
+    substituteInPlace include/radiotray-ng/common.hpp \
+      --replace /usr $out
+  '';
+
+  enableParallelBuilding = true;
+
+  doCheck = true;
+
+  checkPhase = "ctest";
+
+  preFixup = ''
+    gappsWrapperArgs+=(
```

Comment:
The following two lines are already part of [wrapGAppsHook](https://github.com/NixOS/nixpkgs/blob/b14db2cddfb054c0ff46523fa95ba56153b230e0/pkgs/build-support/setup-hooks/wrap-gapps-hook.sh).

Reply:
Oops, I meant to request changes.

Reply:
Good call, thanks! Updated to remove the redundant bits.